app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js

```diff
@@ -213,6 +213,8 @@ ManageIQ.angular.app.controller('cloudVolumeFormController', ['miqService', 'API
     // model attribute with AWS.
     vm.cloudVolumeModel.aws_volume_type = data.volume_type;
     vm.cloudVolumeModel.aws_availability_zone_id = data.availability_zone.ems_ref;
+    vm.cloudVolumeModel.aws_encryption = data.encrypted;
+    vm.cloudVolumeModel.aws_iops = data.iops;
```

Comment:
This looks OK.
However, do we now need to delete the code where we calculate the `aws_iops` value - https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/assets/javascripts/controllers/cloud_volume/cloud_volume_form_controller.js#L130 ?